src/datumaro/components/task.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (C) 2024 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from __future__ import annotations
+
+from enum import IntEnum
+from typing import Mapping, Set
+
+from datumaro.components.annotation import AnnotationType
+
+
+class TaskType(IntEnum):
+    unknown = 0
+    classification = 1
+    classification_multilabel = 2
+    classification_hierarchical = 3
+    detection = 4
+    detection_landmark = 5
+    detection_rotated = 6
+    detection_3d = 7
+    segmentation_semantic = 8
+    segmentation_instance = 9
+    segmentation_panoptic = 10
+    caption = 11
+    super_resolution = 12
+    depth_estimation = 13
+    mixed = 14
```

Comment:
Is there a way for users to know what tasks are possible when TaskType is mixed?

Reply:
`Mixed` task can be transformed to any task types. The reason why we are providing `mixed` is because Datumaro format can have any `AnnotationType` when importing.